Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -809,17 +843,38 @@ void ACesium3DTileset::LoadTileset() {
 
   ACesiumCreditSystem* pCreditSystem = this->ResolveCreditSystem();
 
+  UCesiumBoundingVolumePoolComponent* pBoundingVolumePool =
+      this->FindComponentByClass<UCesiumBoundingVolumePoolComponent>();
+  if (this->EnableOcclusionCulling && !pBoundingVolumePool) {
+    const glm::dmat4& cesiumToUnreal =
+        GetCesiumTilesetToUnrealRelativeWorldTransform();
+    pBoundingVolumePool = NewObject<UCesiumBoundingVolumePoolComponent>(this);
+    pBoundingVolumePool->RegisterComponent();
+    pBoundingVolumePool->SetUsingAbsoluteLocation(true);
+    pBoundingVolumePool->SetFlags(
+        RF_Transient | RF_DuplicateTransient | RF_TextExportTransient);
+    pBoundingVolumePool->RegisterComponent();
+    pBoundingVolumePool->UpdateTransformFromCesium(cesiumToUnreal);
+  }
+
   Cesium3DTilesSelection::TilesetExternals externals{
       pAssetAccessor,
       std::make_shared<UnrealResourcePreparer>(this),
       asyncSystem,
       pCreditSystem ? pCreditSystem->GetExternalCreditSystem() : nullptr,
-      spdlog::default_logger()};
+      spdlog::default_logger(),
+      (this->EnableOcclusionCulling && pBoundingVolumePool)
+          ? pBoundingVolumePool->getPool()
+          : nullptr};
 
   this->_startTime = std::chrono::high_resolution_clock::now();
 
   Cesium3DTilesSelection::TilesetOptions options;
 
+  options.enableOcclusionCulling = this->EnableOcclusionCulling;
+  options.occlusionPoolSize = static_cast<uint32_t>(this->OcclusionPoolSize);
```

Comment:
The only reason `ACesium3DTileset::OcclusionPoolSize` is not unsigned is because unsigned integers aren't supported as UProperties. In `TilesetOptions` on the other hand, we use uint32_t for several things that shouldn't be negative, so it feels more consistent to keep it as uint32_t right? Is there a good reason to make this one int32_t?